README.md

```diff
@@ -1,6 +1,17 @@
 API Application for camptocamp.org v6
 =====================================
 
+Requirements
+------------
+
+ * Python 3.4
+ * PostgreSQL 9.4 / PostGIS 2.1
+ * virtualenv (1.7 or higher)
```

Comment:
For redis: `redis-server`.
And for ElasticSearch the DEB can be downloaded from here: https://www.elastic.co/downloads/past-releases/elasticsearch-2-3-2